### ISSUE TITLE
vitalium LS2 ui wont display

### ISSUE THREAD

**Issue by solipsvs** [KXStudio/Repository#277]
on rpi4 reaper.  says it requires openGL v1.4 to display.  any advice for a no0bish musician on how to get this running?  love all the plugins, this is the first one i tried.

thanks

**Comment by solipsvs** [comment]
![vitalium](https://user-images.githubusercontent.com/93066067/150870253-1e1016ff-7fcf-4aaf-9d36-cadcabddfb9c.jpg)

**Comment by falkTX** [comment]
Yeah vitalium requires a little fancy hardware graphics support, it doesnt run on all systems.
You can use software rendering by setting `export LIBGL_ALWAYS_SOFTWARE=1` env var before running the DAW. UI will be sluggish but should appear correctly.

**Comment by solipsvs** [comment]
> Yeah vitalium requires a little fancy hardware graphics support, it doesnt run on all systems. You can use software rendering by setting `export LIBGL_ALWAYS_SOFTWARE=1` env var before running the DAW. UI will be sluggish but should appear correctly.

thanks for the reply!  i did try the command you listed but it didnt do anything really.  should i add the ‘env var’ as well?  are you running vitalium in reaper on rpi4?  any other suggestions?  id really love to use it

**Comment by solipsvs** [comment]
oh and does the vst use openGL 1.4 too or will it work easier?  reaper doesnt do vst2 so i would need to use another host

**Comment by falkTX** [comment]
Doing a bit of cleanup, issue was related to plugin required GL, nothing to do in terms of packaging. Closing.
